dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -20,8 +20,8 @@
 
     public static Object normaliseValue(Object currVal) throws SQLException {        
         if (currVal==null) return null;
-        TypeNormaliser tn=TypeNormaliserFactory.getNormaliser(currVal.getClass());
-        if (tn!=null) currVal=tn.normalise(currVal);
+        TypeTransformer tn=TypeNormaliserFactory.getNormaliser(currVal.getClass());
+        if (tn!=null) currVal=tn.transform(currVal);
```

Comment:
As we're anyway touching these lines, let's reformat and add space around `=` and `!=`.